### PR TITLE
Mention admin and how it pertains to limiting supply

### PR DESCRIPTION
**Pull request by sisuresh**
If there are any other docs on locking the issuing account, these same updates will need to be made there as well.

### CODE REVIEW

File: docs/issuing-assets/control-asset-access.mdx [hunk 1]
```diff
@@ -220,7 +220,7 @@ except BaseHorizonError as e:
 
 With that warning in mind:
 
-It is possible to lock down the issuing account of an asset so that the asset’s supply is permanently fixed. To do this, set the issuing account’s master weight to 0 using the Set Options operation. This prevents the issuing account from being able to sign transactions and therefore, is unable to issue any more assets. Be sure to do this only after you’ve issued all desired assets to the distribution account.
+It is possible to lock down the issuing account of an asset so that the asset’s supply cannot increase. To do this, first set the issuing account’s master weight to 0 using the Set Options operation. This prevents the issuing account from being able to sign transactions and therefore, making the issuer unable to issue any more assets. Be sure to do this only after you’ve issued all desired assets to the distribution account. Then make sure that the admin for the Stellar Asset Contract for this asset was not updated from the default (which is the issuer) using the `set_admin` contract call. If the admin was not the issuer, then the admin would be able to mint the asset even with the issuing account locked.
```
Comment (briwylde08) [inline docs/issuing-assets/control-asset-access.mdx:223]: Not every Stellar asset has a SAC, correct? Someone has to actually deploy one for that asset? Would it make sense to say something like: "If the asset has a SAC, make sure the admin for the SAC was not updated..."

Comment (sisuresh) [reply docs/issuing-assets/control-asset-access.mdx]: Yeah that's a good point. Fixed.

**Comment by briwylde08** [comment]
Doing a little audit, the other places that mention locking your account are:
https://developers.stellar.org/docs/learn/encyclopedia/signatures-multisig#thresholds
https://developers.stellar.org/docs/building-apps/wallet/stellar#modify-account
https://developers.stellar.org/docs/learn/encyclopedia/signatures-multisig#example-4-company-accounts

I'm happy to take a stab at editing these but might need some help with language.

**Comment by sisuresh** [comment]
> Doing a little audit, the other places that mention locking your account are: https://developers.stellar.org/docs/learn/encyclopedia/signatures-multisig#thresholds https://developers.stellar.org/docs/building-apps/wallet/stellar#modify-account https://developers.stellar.org/docs/learn/encyclopedia/signatures-multisig#example-4-company-accounts
> 
> I'm happy to take a stab at editing these but might need some help with language.

I think my statement about updating all docs that mention locking the account was too broad. In the linked instances, there's no mention of assets, so I think it's fine to leave as is? The important point to get across is that a locked issuer alone is not enough to guarantee a limited supply, which this PR now mentions in this section about limiting supply. What do you think?

**Comment by briwylde08** [comment]
My immediate thought was the SAC probably didn't need to be mentioned in those instances but thought I'd point to them just in case! So I also think it's fine to leave as is.

**Review by briwylde08 — APPROVED** [review]

**Pull request merged**